--- dbus-glib-0.71/configure.in.orig	2006-07-24 20:41:39.000000000 +0200
+++ dbus-glib-0.71/configure.in	2006-08-20 12:21:11.383783250 +0200
@@ -400,8 +400,8 @@
 fi
 
 #### Various functions
-AC_CHECK_LIB(socket,socket)
-AC_CHECK_LIB(nsl,gethostbyname)
+AC_SEARCH_LIBS(socket,socket)
+AC_SEARCH_LIBS(gethostbyname,nsl)
 
 AC_CHECK_FUNCS(vsnprintf vasprintf nanosleep usleep poll setenv unsetenv socketpair getgrouplist)
 
